packages/terra-form-radio/tests/wdio/radio-spec.js

```diff
@@ -0,0 +1,171 @@
+        '--terra-form-radio-hover-border-color': '#800080',
+        '--terra-form-radio-hover-border-width': '2px',
+      });
+    });
+
+    describe('Default Custom Radio - Checked', () => {
+      beforeEach(() => {
+        browser.waitForVisible('#default');
+        browser.click('#default');
+      });
+
+      Terra.should.matchScreenshot();
+      Terra.should.beAccessible();
+
+      Terra.should.themeEachCustomProperty({
+        '--terra-form-radio-outer-ring-checked-background-color': '#fff',
+        '--terra-form-radio-outer-ring-checked-border-color': '#4a90e2',
+        '--terra-form-radio-outer-ring-checked-border-radius': '50%',
+        '--terra-form-radio-outer-ring-checked-border-style': 'solid',
+        '--terra-form-radio-outer-ring-checked-border-width': '1px',
+        '--terra-form-radio-outer-ring-checked-height': '20px',
+        '--terra-form-radio-outer-ring-checked-margin-top': '4px',
+        '--terra-form-radio-outer-ring-checked-width': '20px',
+        '--terra-form-radio-inner-ring-checked-background-color': 'rgb(255, 0, 0)',
+        '--terra-form-radio-inner-ring-checked-border-color': 'rgb(255, 0, 0)',
+        '--terra-form-radio-inner-ring-checked-border-style': 'solid',
+        '--terra-form-radio-inner-ring-checked-border-width': '1px',
+        '--terra-form-radio-inner-ring-checked-height': '14px',
+        '--terra-form-radio-inner-ring-checked-transition': 'all 0.3s cubic-bezier(0.78, 0.14, 0.15, 0.86)',
+        '--terra-form-radio-inner-ring-checked-width': '14px',
+      });
+    });
+
+    describe('Default Custom Radio - Focus', () => {
+      beforeEach(() => {
+        browser.waitForVisible('#default');
+        browser.click('#default');
+      });
+
+      Terra.should.matchScreenshot();
+      Terra.should.beAccessible();
+
+      Terra.should.themeEachCustomProperty({
+        '--terra-form-radio-focus-border-color': '#88a8ef',
+        '--terra-form-radio-focus-border-width': '2px',
+      });
+    });
+
+    describe('Default Custom Radio - Focus Ring', () => {
+      beforeEach(() => {
+        browser.waitForVisible('#default');
+        browser.moveToObject('#default');
+      });
+
+      Terra.should.matchScreenshot();
+      Terra.should.beAccessible();
+
+      Terra.should.themeEachCustomProperty({
+        '--terra-form-radio-focus-ring-background-color': 'rgba(74, 144, 226, 0.25)',
+        '--terra-form-radio-focus-ring-height': '30px',
+        '--terra-form-radio-focus-ring-left': '-5px',
+        '--terra-form-radio-focus-ring-top': '-5px',
+        '--terra-form-radio-focus-ring-width': '30px',
+      });
+    });
+  });
+
+  describe('Disabled Custom Radio', () => {
+    beforeEach(() => {
+      browser.url('/#/tests/form-radio-tests/disabled');
+    });
+
+    Terra.should.matchScreenshot();
+    Terra.should.beAccessible();
+
+    Terra.should.themeEachCustomProperty({
+      '--terra-form-radio-disabled-font-color': '#9b9fa1',
```

Comment:
These themeable variables should be testing non-default values. In the stylesheet, you define the disabled-font-color to be
> color: var(--terra-form-radio-disabled-font-color, #9b9fa1);

Here you want to verify the theming variable works with a new value for the screenshot comparison.

Reply:
Yeah I forgot to change that one, I'll change it 👍